components/brave_wallet_ui/page/screens/composer_ui/from_asset/from_asset.style.ts

```diff
@@ -28,6 +29,7 @@ export const FromText = styled(Text)`
 export const BalanceText = styled(Text)`
   line-height: 26px;
   color: ${leo.color.text.primary};
+  margin-right: 4px;
```

Comment:
can we set a gap on the row that contains this instead?

Reply:
Same for this case!